examples/data_preprocessing/wiennerisches_diarium_historical_dataset/Sterbelisten_Preprocessing.ipynb

```diff
@@ -0,0 +1,395 @@
+   "execution_count": 117,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(6, 2)"
+      ]
+     },
+     "execution_count": 117,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "'''\n",
+    "T Matrix\n",
+    "Dimensions keywords x 2\n",
+    "\n",
+    "'''\n",
+    "import numpy as np\n",
+    "a= np.zeros(len(locations))\n",
+    "b= np.full(len(locations),1)\n",
+    "\n",
+    "T = np.stack((b,a.T))\n",
+    "T.T.shape\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Z Matrices**\n",
+    "Each sentense is going to be its own Z Matrix.\n",
+    "colums = place names\n",
+    "rows = words in sentense"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 119,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "'''\n",
+    "Here I am building the Z Matrixes by looping through all samples.\n",
+    "Each sample is compared with the list of locations, if the location is in the sample,\n",
+    "the new list \"matched_loc\" gets an entry 1 otherwise a 0 is added\n",
+    "Then the list is transformed to an numpy array and brought into shape:\n",
+    "number of place names x number of words\n",
+    "\n",
+    "'''\n",
+    "locations_list = list(range(0,(len(locations))))\n",
+    "\n",
+    "\n",
+    "for sample in samples:\n",
+    "    matched_loc= [1 if i == j else 0 for i in sample for j in locations_list]\n",
+    "    i=len(sample)\n",
+    "    Z = np.array(matched_loc)\n",
+    "    Z = np.reshape(Z, (i,len(locations_list)))\n",
+    "# I haven't found a way to save them with their index number! "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 120,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'create a tokenized sentenses with spacy'"
+      ]
+     },
+     "execution_count": 120,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "\"\"\"create a tokenized sentenses with spacy\"\"\"\n",
+    "#import spacy \n",
+    "#from spacy.lang.de.examples import sentences\n",
+    "\n",
+    "#nlp = spacy.load(\"de_core_news_sm\")\n",
+    "\n",
+    "#for i in sterbeliste:\n",
+    "    #doc = nlp(i)\n",
+    "    #print(doc.text)\n",
+    "    #for token in doc:\n",
+    "        #print(token.text, token.pos_, token.dep_)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 5. Building test data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "'''Loading manually checked test data'''\n",
+    "\n",
+    "with jsl.open('timemachine_evaluation_v1_edited_corrected.jsonl','r') as reader: #read in jsonl with jsonlines\n",
+    "    df = pd.json_normalize(reader) \n",
+    "\n",
+    "### I don't understand where the tagged toponym is or whether there is any"
```

Comment:
we can discuss it on Monday as well

Reply:
Yes, I added an explanation for why this data is not suitable as test data for our task